### PR TITLE
Feature/systems and trees overview

### PR DESCRIPTION
**Pull request by Kasmadei**
Added a table and cards view.
Resolves partially: https://github.com/kbss-cvut/fta-fmea-ui/issues/247

### CODE REVIEW

**Comment by blcham** [comment]
Please do "docker-compose pull". There are new data to be shown in the list from this PR:
https://github.com/kbss-cvut/fta-fmea/pull/84

image

**Comment by blcham** [comment]
Merging, although the issue is not resolved as there are new data that can be shown there.

File: public/locales/cs/translation.json [hunk 1]
```diff
@@ -68,5 +68,17 @@
       "title": "Manuálně definovaná intenzita poruch",
       "description": "Intenzita poruch nastavená manuálně týmem AVA na základě zkušeného inženýrského úsudku."
     }
+  },
+  "faultTreeOverviewTable": {
+    "name": "Jméno",
+    "aircraftType": "Typ letadla",
+    "ata": "ATA",
+    "calculatedFailureRate": "Vypočtená int. poruch",
+    "fhaBasedFailureRate": "Založená int. poruch",
```
Comment (grotskat) [inline public/locales/cs/translation.json:77]: Int. poruch založená na FHA
@blcham

Comment (blcham) [reply public/locales/cs/translation.json:77]: @grotskat this PR is already merged and we need to review all translations. Please resolve it in the issue https://github.com/kbss-cvut/fta-fmea-ui/issues/261

**Comment by blcham** [comment]
Strange that when I run it locally i get:
image

**Pull request merged**